specs/tendrl_host_listing_summary_api.adoc

```diff
@@ -0,0 +1,182 @@
+// vim: tw=79
+
+= Host Listing Backend Support
+
+== Introduction
+
+The host listing page displays the following details per host:
+
+* Hostname
+* Storage used, total and percentage used
+* Cpu percentage used
+* Memory used, total and percentage used
+* Name of Cluster to which host belongs
+* Role of the node in the cluster
+* Alerts counter that drills down to alerts list filtered by hostname.
+
+The mock up of the UI can be found at:
+https://redhat.invisionapp.com/share/BR8JDCGSQ#/screens/199053039
+
+== Problem description
+
+All the details specified in the section under "Introduction" section are
+available at different locations like the following:
+
+* The hostname, Name of Cluster to which host belongs, Role of the node in the
+  cluster and alerts each in etcd.
+* The cpu, memory and storage usage details in time-series db.
+
+This necessiates either the UI to fetch each of the details from different
+sources by making multiple api calls or the tendrl-api looking at multiple
+sources for these information.
+
+== Use Cases
+
+* Host listing page in tendrl UI.
+
+== Proposed change
+
+* The performance-monitoring application anyway performs task of aggregating
+  statistics in time-series db for aggregations at cluster and system levels.
+  At the same time, it can push the host level details to etcd to the path
+  '/monitoring/nodes/node-id/summary'.
+* Whenever a new alert that is related to the node is detected in the alerting
```

Comment:
Should we talk about alert clearance as well well. That would cause decrements in no of alerts as well I feel. Need more detail I feel